apps/storybook/src/stories/onchain-info/TokenChip.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { withWagmiProvider } from "../decorators/wagmi";
+import { TokenChip } from "./TokenChip";
+
+const meta = {
+  title: "OnchainInfo/TokenChip",
+  component: TokenChip,
+  parameters: {
+    layout: "centered",
+  },
+  args: {},
+  decorators: [withWagmiProvider()],
+} satisfies Meta<typeof TokenChip>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const ETHTokenChip: Story = {
+  args: {
+    image:
+      "https://ethereum.org/_next/image/?url=%2F_next%2Fstatic%2Fmedia%2Feth-diamond-black.a042df77.png&w=828&q=75",
```

Comment:
lets curate a fixture of images of tokens

we might find those on ipfs / upload ourselves

Reply:
Oke ser

Reply:
this will be for later as per discussed